proxy/src/telemetry/metrics/prometheus/labels.rs

```diff
@@ -2,18 +2,26 @@
 use http;
 use std::fmt;
 
+use control::discovery;
 use ctx;
 
-#[derive(Clone, Debug, Default, Eq, PartialEq, Hash)]
+#[derive(Clone, Debug, Eq, PartialEq, Hash)]
 pub struct RequestLabels {
 
-    outbound_labels: Option<OutboundLabels>,
+    /// Was the request in the inbound or outbound direction?
+    direction: Direction,
+
+    // Additional labels identifying the destination service of an outbound
+    // request, provided by the Conduit control plane's service discovery.
+    outbound_labels: Option<discovery::DstLabels>,
 
     /// The value of the `:authority` (HTTP/2) or `Host` (HTTP/1.1) header of
     /// the request.
     authority: String,
+
```

Comment:
Could you please remove all the extra newlines that were added in this PR? They are distracting. There's no need for a newline before or after a brace.

Reply:
This newline should be removed.

Reply:
Sorry! I recall @olix0r saying at one point that he prefers newlines around struct fields, so I was just trying to conform to that.